improver_tests/utilities/test_mathematical_operations.py

```diff
@@ -17,7 +15,10 @@
     set_up_variable_cube,
 )
 from improver.utilities.cube_manipulation import sort_coord_in_cube
-from improver.utilities.mathematical_operations import Integration, fast_linear_fit
+from improver.utilities.mathematical_operations import (
```

Comment:
All changes to this file can be reverted, as we've moved our tests elsewhere, so shouldn't be modifying this.